sops/amq-scaling.md

```diff
@@ -0,0 +1,46 @@
+# AMQ Online Scaling Guide
+The purpose of this guide is to outline the existing scaling capabilities of AMQ Online, including scaling options for high availability and performance driven implementations along with some known limitations within the product.
+
+
+## AMQ Online Components 
+The basic AMQ Online infrastructure consists of the following components: 
+ - Operator
+ - Address-Space-Controller 
+ - Api-server,
+ - Console
+ - (optional) auth-service.
+
+ There is no scaling options for the above components.
+- Controller
+- Console
+- Brokers
+- Routers
+
+While these components above do not scale in the traditional sense. Those components that do scale are the `Routers` and `Brokers` as they experience the most load during use. Typically scaled horizontally and will scale automatically.
```

Comment:
Seems to be a 'fullstop' mid sentence here. Could reword as:

"The `Router` and `Broker` components tend to experience the most load during use. These are typically scaled horizontally rather than vertically and are scaled automatically by the ........?"

Reply:
@ciaranRoche Is this sentence needed now? "While these components above do not scale in the traditional sense."